addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.xml

```diff
@@ -11,30 +11,17 @@
     </t>
 
     <t t-name="mail.DiscussSidebarCategory">
-        <div class="o-mail-DiscussSidebarCategory d-flex align-items-center my-1" t-att-class="category.extraClass">
-            <button class="btn btn-link text-reset flex-grow-1 d-flex align-items-baseline mx-1 p-0 text-start opacity-100-hover opacity-75" t-on-click="() => this.toggleCategory(category)">
+        <div class="o-mail-DiscussSidebarCategory d-flex align-items-center my-1" t-att-class="category.extraClass" t-ref="categoryHeader">
+            <button class="btn btn-link text-reset flex-grow-1 d-flex align-items-baseline mx-1 p-0 text-start opacity-100-hover opacity-75" t-on-click="() => this.toggleCategory(category)" t-ref="openCategoryBtn">
```

Comment:
The template is named `DiscussSidebarCategory`, using `t-ref`/`name` that has "category" in the name is redundant.

I would rename `categoryHeader` and `openCategoryBtn` to respectively `header` and `toggler`.